app/src/prod/java/com/austindroids/austinfeedsme/di/modules/DataModule.java

```diff
@@ -38,7 +32,7 @@ EventsRepository eventsRepository(FirebaseEventsDataSource firebaseEventsDataSou
 
     @Provides @Named("meetup") @Singleton
     EventsDataSource meetupDataSource() {
-        return new MeetupDataSource(context);
+        return new MeetupDataSource();
```

Comment:
This can somewhat be replaced with `@Inject` constructor, but then it'd need to be `@Binds`.

Reply:
Not exactly sure how to do this. Dagger 2 wanted me to convert the class to abstract and I was curious if this would have negative impact on anything else.

Reply:
This helped: https://google.github.io/dagger/api/latest/dagger/Binds.html

and this: 

https://stackoverflow.com/questions/43287645/dagger-2-injecting-constructors

Reply:
I get the following "@Qualifier annotations are not allowed on @Inject constructors."